chromium_src/ui/views/controls/button/md_text_button.cc

```diff
@@ -295,6 +289,9 @@ void MdTextButton::UpdateColorsForBrave() {
   SkColor bg_color = GetBgColorOverride().value_or(
       style.background_color.value_or(SK_ColorTRANSPARENT));
   SkColor stroke_color = style.border_color.value_or(SK_ColorTRANSPARENT);
+
+  // SubPixelRendering doesn't work if we have any background opacity.
+  SetTextSubpixelRenderingEnabled(opacity == 1 && SkColorGetA(bg_color) == 0xFF);
```

Comment:
Oh, nice fix! 👍🏼 
And As you said in the comment, `opacity` value seems not important factor.
IMO , `SetTextSubpixelRenderingEnabled(SkColorGetA(bg_color) == 0xFF);`  seems more exact. WDYT?

Reply:
I think we need it because on line 295 we blend in that much opacity. So the bg color needs to have no transparency, which is only true when opacity is 1 and bg_color has 0xFF in it's alpha channel.

Reply:
Got it :)